cashctrl_ledger/ledger.py

```diff
@@ -197,6 +203,36 @@ def settings_clear(self):
     # ----------------------------------------------------------------------
     # Accounts
 
+    def sanitize_account_groups(self, groups: pd.Series) -> pd.Series:
+        """Ensure account group paths are sanitized with a leading slash and a valid root node.
+
+        Args:
+            groups (pd.Series): A pandas Series containing account group paths.
+
+        Returns:
+            pd.Series: Sanitized account group series.
+        """
+        # Remove leading slashes
```

Comment:
Please drop the comment. (Better to treat all REGEX patterns alike, no need for special treatment for the first one.)